data-raw/mario_kart_auction.R

```diff
@@ -0,0 +1,5 @@
+## code to prepare `mario_kart_auction` dataset goes here
+library(tidyverse)
+mario_kart_auction <- "https://www.openintro.org/data/csv/mariokart.csv" %>%
```

Comment:
The variables in the dataset, `condition`, `ship_sp`, and `stock_photo` should be of type factor, not character.

Reply:
Also, documentation is missing.